src/services/appointment.ts

```diff
@@ -0,0 +1,21 @@
+import { Op } from 'sequelize';
+import { Appointment } from '../models';
+import sequelize from '../db/connection';
+
+const appointmentQuery = async (therapistId: string, date: any = 'today') => {

```

Comment:
this is not query (sequleize is not a query language), rename please to `getAppointmentsPerDateService`